src/components/Header/style.jsx

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+
+const SHeader = styled.div`
+  .header {
```

Comment:
No need for a class "header" since you'll target the header tag with styled.header